src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__box calendar__box--month-lenght--31">
+        <div class="calendar__cell calendar__cell--start-day--mon"></div>
```

Comment:
`calendar__cell--start-day--mon` it should be modifier of the whole calendar, not first cell
but logic is right